### PR TITLE
layout and tests

### PR DESCRIPTION
**Pull request by chrxs**

### CODE REVIEW

File: src/views/MonthView.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import DateHeader from 'components/DateHeader'
+
+const MonthView = (props) => {
+  const { params } = props.match
+  console.log('MonthView', params)
```
Comment (bradjenn) [inline src/views/MonthView.js]: you keeping this?

File: src/views/YearView.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import DateHeader from 'components/DateHeader'
+
+const YearView = (props) => {
+  const { params } = props.match
+  console.log('YearView', params)
```
Comment (bradjenn) [inline src/views/YearView.js]: and this?

File: src/views/MonthView.js (hunk 1 above)
Comment (chrxs) [inline src/views/MonthView.js]: yeah, for now

File: src/views/YearView.js (hunk 2 above)
Comment (chrxs) [inline src/views/YearView.js]: yeah

**Pull request merged**